public/content/en/basics/delegates.md

```diff
@@ -41,16 +45,25 @@ standard function
 [`std.functional.toDelegate`](https://dlang.org/phobos/std_functional.html#.toDelegate)
 converts a `function` to a `delegate`.
 
-Nameless function which are called *lambdas* can be defined in two ways:
+### Anonymous functions & Lambdas
+
+As functions can be saved as variables and passed to other functions,
+it is laborious to give them an own name and to define them. Hence D allows
+nameless functions and one-line _lambdas_.
 
     auto f = (int lhs, int rhs) {
         return lhs + rhs;
     };
-    auto f = (int lhs, int rhs) => lhs + rhs;
+    auto f = (int lhs, int rhs) => lhs + rhs; // Lambda - internally converted to the above
+
+It is also possible to pass-in strings as template argument to functional parts
+of D's standard library. For example they offer a convenient way
+to define a folding (aka reducer):
+
+    [1, 2, 3].reduce!`a + b`; // 6
```

Comment:
I am not sure yet whether explaining string functions is a good idea, but they appear often in D code and I couldn't find a better place.